metadata/test/src/main.rs

```diff
@@ -0,0 +1,118 @@
+use solana_client::rpc_client::RpcClient;
+use solana_program::program_pack::Pack;
+use solana_sdk::{
+    pubkey::Pubkey,
+    signature::{read_keypair_file, Signer},
+    system_instruction::create_account,
+    transaction::Transaction,
+};
+use spl_metadata::{
+    instruction::{create_metadata_accounts, init_metadata_accounts, update_metadata_accounts},
+    state::metadata::{Metadata, NAME_LENGTH, SYMBOL_LENGTH},
+    state::PREFIX,
+};
+use spl_token::{instruction::initialize_mint, state::Mint};
+use std::str::FromStr;
```

Comment:
```
use {
  .
  .
  .
};
``` 
please

Reply:
I dont understand

Reply:
sorry, I'm generally pushing for this style whenever I touch new code:
```
use {
   solana_client::rpc_client::RpcClient,
   solana_program::program_pack::Pack,
};
```
versus
```
use solana_client::rpc_client::RpcClient;
use solana_program::program_pack::Pack;
```

Reply:
I wish clippy would  just yell about it

Reply:
Ah, ok. I understand. Will do.